.travis.yml

```diff
@@ -8,8 +8,6 @@ branches:
 
 matrix:
   include:
-    - python: 3.5
```

Comment:
Why are we removing support for python 3.5? Is such is the case we have to update the README as well.

Reply:
Type hints are heavily used in this project, but they're not supported by Python 3.5 (the way they're used here). Instead of refactoring, it seems like more reasonable to ditch Python 3.5. It also doesn't offer support for f-strings. I'm opening another pull request to make this change alone.

Reply:
I have removed the Python 3.5 support because of usages of Type Hints in the `autoextract.aio` module were not supported by this version.

I can see in the README file that we're offering support for basic / synchronous calls with Python 3.5. This is weird to because part of the codebase supports a given version of Python while other doesn't. Shouldn't we remove the Python 3.5 support?

Python 3.5 could become difficult to maintain regarding the limitations with Type Hints and they also don't support f-strings.

I guess it should be done in another pull request, tho. I'll try to re-enable Python 3.5 pipelines and skip the incompatible test cases.